kore/src/Kore/Exec.hs

```diff
@@ -41,6 +43,9 @@ import Control.Monad.Trans.Except
 import Data.Coerce
     ( coerce
     )
+import Data.Functor
+    ( (<&>)
+    )
```

Comment:
Is there any set criteria for what belongs in `Prelude.Kore`?  I see this is imported elsewhere in the repo so perhaps this comment isn't appropriate for this specific pull request, but I would have suspected that `<&>` is simple/useful enough to be placed in the prelude.

Reply:
I don't think we have rigorous criteria for `Prelude.Kore`. I'll ask Tom about `<&>`